integration/app/list/list.component.html

```diff
@@ -4,5 +4,9 @@
 </div>
 
 <div *ngIf="router$ | async as router">
-  animals were resolved {{ router.root.firstChild.firstChild.data.list }}
+  {{ join('router$', router.root.firstChild.firstChild.data.list) }}
+</div>
+
+<div *ngIf="route$ | async as route">
+  {{ join('route$', route.data.list) }}
```

Comment:
Why function instead pipe? Usage function in template is bad practice

Reply:
This is a div just to check that the list was resolved, I don't see any practical benefit to create a pipe specially in this case.

Reply:
As you know, but such code in the template makes me confused

Reply:
I'm not sure what exactly makes you confused and I'm also not sure if the method should be a blocker in this case :wink: I'm not using this code within our library, this code is used only for SSR tests, what would be the difference between method and pipe? Would like to hear your thoughts.

Reply:
Maybe a better method name would do, like `getListDisplayText`. I was confused by `join`, thinking that it was `string.join`.

Reply:
@markwhitfeld Okay, NP